app/controllers/wikis_controller.rb

```diff
@@ -0,0 +1,59 @@
+class WikisController < ApplicationController
+  def index
+    @wikis = Wiki.all
+  end
+
+  def show
+    @wiki = Wiki.find(params[:id])
+  end
+
+  def new
+    @wiki = Wiki.new
+  end
+
+  def create
+    @wiki = Wiki.new
+    @wiki.title = params[:wiki][:title]
+    @wiki.body = params[:wiki][:body]
+    @wiki.private = params[:wiki][:private]
+
```

Comment:
Still perfectly acceptable code, but remind me to discuss strong params with you on our next call